.github/workflows/kogito-apps-pr.yml

```
@@ -28,24 +28,25 @@ jobs:
     name: Maven Build
     steps:
       - name: Clean Disk Space
-        uses: kiegroup/kogito-pipelines/.ci/actions/ubuntu-disk-space@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/ubuntu-disk-space@main
         if: ${{ matrix.os == 'ubuntu-latest' }}
       - name: Support long paths
         if: ${{ matrix.os == 'windows-latest' }}
-        uses: kiegroup/kogito-pipelines/.ci/actions/long-paths@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/long-paths@main
       - name: Java and Maven Setup
-        uses: kiegroup/kogito-pipelines/.ci/actions/maven@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
+          definition-file: 'https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml'
```

mmm the GHA keeps failing because it seems that env variable `BRANCH` is not defined, therefore from `'https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml'` it taked the definition file from `main` branch. Therefore I would say this issue was here also before.

To fix this we could set the default branch to `1.13.x` instead of `main`, @Ginxo do you remember how it was supposed to be exported that `BRANCH` env variable? is it beacuse we use action from `kie-ci:main` branch?

```suggestion
          definition-file: 'https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:1.13.x}/.ci/pull-request-config.yaml'
```

---

yeah that is clear, what I do not get is why using previous configuration `kogito-pipelines@1.13.x` was correctly taking the definition file from `1.13.x` branch (I assume that `BRANCH=1.13.x`) wheeras with this new one it seems it is taking def file from `main` branch (so either `BRANCH=main` or `BRANCH` is nto defined at all)

---

Guys, how can we proceed with this? @lampajr @Ginxo

---

IMO as `kogito-apps` is no more maintained on `1.13.x` branch I think we can also close this PR? wdyt? otherwise my suggestion to set the default branch to 1.13.x should work but the PR check will fail anyway as kogito-apps is removed/comment there https://github.com/kiegroup/kogito-pipelines/blob/1.13.x/.ci/project-dependencies.yaml#L34-L49 so..

---

I changed back the default branch to 1.13.x, but we can close this PR if everybody agrees. It is ok for me.

---

@Ginxo wdyt?

---

I would merge it as it is, please proceed if you agree guys, otherwise close it